app/models/git/track.rb

```diff
@@ -42,6 +42,8 @@ def commit
     private
     attr_reader :repo, :git_sha
 
-    delegate :read_text_blob, :read_json_blob, to: :repo
+    def full_filepath(filepath)
```

Comment:
I suggest this should be called `expand_filepath`

Reply:
I just renamed it to `absolute_filepath`, which was also how it was described somewhere else in the source code.